clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -134,10 +140,10 @@ public boolean containsKey(final K key) throws StoreAccessException {
   @Override
   public PutStatus put(final K key, final V value) throws StoreAccessException {
     putObserver.begin();
-    PutOperation<K, V> operation = new PutOperation<K, V>(key, value);
+    PutOperation<K, V> operation = new PutOperation<K, V>(key, value, timeSource.getTimeMillis(), true);
```

Comment:
An additional helper constructor in the operation classes that would handle the _isFirst_ flag implicitly would have been nice.